Rakefile

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+require 'rake'
+require 'rake/testtask'
+require 'bundler/gem_tasks'
+
+desc 'Default: run unit tests.'
+task :default => :test
+
+desc 'Test Salt plugin'
+Rake::TestTask.new(:test) do |t|
+  t.libs << '.'
+  t.libs << 'lib'
+  t.libs << 'test'
+  t.test_files = FileList['test/**/*_test.rb']
+  t.verbose = true
+end
+
+require 'rubocop/rake_task'
+
+desc 'Run RuboCop on the lib directory'
+RuboCop::RakeTask.new(:rubocop) do |task|
+  task.patterns = ['lib/**/*.rb', 'test/**/*.rb']
+  task.fail_on_error = false
```

Comment:
Why don't you want this?

Reply:
You mean the "fail_on_error"? Copied from smart_proxy_ansible